Resources/Private/Language/locallang.xlf

```diff
@@ -6,6 +6,9 @@
 			<trans-unit id="config.backendPreview">
 				<source>Add backend preview templates: If enabled the export contains some PHP and Fluid files to show a record preview in the page layout.</source>
 			</trans-unit>
+			<trans-unit id="config.exportIcons">
+				<source>Export content element icons: If enabled the export contains some PHP to export defined content element icons.</source>
```

Comment:
You only added the feature to export defined fontawsome icons. By the way, is there any check which checks weather the icon was set in mask configuration? Please don't tell that you are exporting content element icons but the fontawsome icons